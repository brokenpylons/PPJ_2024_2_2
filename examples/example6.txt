var a = 1 + 2;
var b = 3;
print(a * b);
