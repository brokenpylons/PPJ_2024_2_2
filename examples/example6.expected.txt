real("2") minus("-") real("1") times("*") real("1") 
